config/freertos/esp32/create.sh

```diff
@@ -3,7 +3,11 @@ pushd $FW_TARGETDIR >/dev/null
     mkdir toolchain
 
     pushd toolchain >/dev/null
-        git clone -b v4.1 --recursive https://github.com/espressif/esp-idf.git
+        git clone -b v4.0.1 --recursive https://github.com/espressif/esp-idf.git
```

Comment:
Is there any reason for using IDF 4.0.1 instead of 4.1?

Reply:
No, I haven't noticed that the IDF version has changed.